ui/src/app/machines/views/MachineDetails/MachineNetwork/NetworkActions/NetworkActions.tsx

```diff
@@ -0,0 +1,57 @@
+import { Button, Col, Row } from "@canonical/react-components";
+import { useSelector } from "react-redux";
+
+import type { SetSelectedAction } from "../../types";
+
+import { useSendAnalytics } from "app/base/hooks";
+import machineSelectors from "app/store/machine/selectors";
+import type { Machine } from "app/store/machine/types";
+import { useIsAllNetworkingDisabled } from "app/store/machine/utils";
+import type { RootState } from "app/store/root/types";
+import { NodeActions } from "app/store/types/node";
+
+type Props = {
+  systemId: Machine["system_id"];
+  setSelectedAction: SetSelectedAction;
+};
+
+const NetworkActions = ({
+  setSelectedAction,
+  systemId,
+}: Props): JSX.Element | null => {
+  const machine = useSelector((state: RootState) =>
+    machineSelectors.getById(state, systemId)
+  );
+  const isAllNetworkingDisabled = useIsAllNetworkingDisabled(machine);
+  const sendAnalytics = useSendAnalytics();
+
+  if (!machine) {
+    return null;
+  }
+
+  return (
+    <Row className="is-shallow">
```

Comment:
This class is only meant for strips, so can probably be removed

Reply:
Yep you're right, it wasn't doing anything! Serves me right for copying the old code.